submissions/Zakkarat/memory-pair-game/stylesheet/main.css

```diff
@@ -64,37 +72,47 @@ html {
 }
 .modal {
   position: fixed;
-  display: none;
   z-index: 1;
-  left: 0;
   top: 0;
-  font-family: Roboto;
+  left: 0;
+
+  display: none;
   justify-content: center;
+
   width: 100%;
   height: 100%;
-  background-color: rgba(0,0,0,0.4);
+
   transition: 1.5s;
+
+  background-color: rgba(0,0,0,0.4);
+
+  font-family: Roboto;
 }
 .modal-content {
   position: fixed;
-  align-items: center;
+
   display: flex;
+  align-items: center;
+  flex-direction: column;
+  justify-content: center;
+
   width: 370px;
   height: 140px;
-  color: white;
   margin-top: -100px;
-  flex-direction: column;
-  justify-content: center;
-  background-color: #032333;
+
+  color: white;
   border-radius: 5px;
+  background-color: #032333;
   box-shadow: 1px 1px 5px 1px #000;
 }
 .modal-up {
   display : flex;
+
   transition: 1.5s;
```

Comment:
you should avoid empty lines if they are not between block(sometime logic block). You have a lot of them, and prettier doesn't fix this issues(it(prettier) handles only 2+ empty rows, I think)

Reply:
Okay, thank you! I will take it into account.